package.json

```diff
@@ -17,7 +17,7 @@
     "bootstrap": "5.1.3",
     "react": "17.0.2",
     "react-dom": "17.0.2",
-    "react-router-dom": "6.2.1",
+    "react-router-dom": "^6.2.1",
```

Comment:
Please anchor deps versions (remove the `^` and then `npm i` again so package-lock.json will be updated accordingly). Please let me know if you don't understand why (here and everywhere else)

Reply:
remove the ^? They are already removed. I don't understand why or how or who what where.

Reply:
There are 2 different ways to install packages:
1. Using an exact version (e.g. `"react-router-dom": "6.2.1"`)
2. Using a minimal version (e.g. `"react-router-dom": "^6.2.1"`)

The problem with option (2) is that you never know what version you have in your `node_modules` (actually you can but bear with me on that). So, potentially, you can have a package that you have never tested, in production.

Option (1) on the other hand ensures you have an exact (tested, hopefully) version of a package you use, locally and in production. In this case, it is `6.2.1` (not `6.7.0` nor `7.0.1`, but `6.2.1`, guaranteed).

Does this make more sense?